bin/release

```diff
@@ -9,23 +9,23 @@
 ###
 set -e
 
-git checkout master
-git pull
+./bin/validate-release
 
-version=$(< VERSION)
+version="$(< VERSION)"
+
+git fetch origin master
+git reset --hard origin/master
 
 printf "RELEASE %s\n" "$version"
 bundle check || bundle install
 rake release
 
-docker build --rm -t codeclimate/codeclimate .
+docker build --rm --tag codeclimate/codeclimate .
 docker push codeclimate/codeclimate:latest
 docker tag codeclimate/codeclimate "codeclimate/codeclimate:$version"
 docker push "codeclimate/codeclimate:$version"
 
 (cd ../homebrew-formulae/ && bin/release "$version")
 
-echo "Be sure to update release notes:"
-echo ""
-echo "  https://github.com/codeclimate/codeclimate/releases/new?tag=v$version"
-echo ""
```

Comment:
The extra whitespace here was intentional. I found I missed this message if it wasn't set off from all the other logging. To address the shellcheck issue but keep the formatting, you could move to this:

```sh
cat <<EOM
Be sure to...

  https://...

EOM
```